packaging/observability/opentelemetry-collector-large.yaml

```diff
@@ -0,0 +1,94 @@
+# This configuration exports:
+# - Container, Pod, Volume, and Node metrics
+# - Kubernetes Events (Warnings only)
+# - Host's CPU, Mem, Disk, and Network Metrics
+# - System journals for selected MicroShift services and dependencies, priority < Info
+
+receivers:
```

Comment:
I wonder if we'd like to shuffle the yamls a bit to make the difference between files more apparent (like sections that differ at the top - `receivers:`, `service:`, and `processors:` and `exporters:` at the bottom because they're the same).

But I don't want to force it, maybe this structure is natural in OTEL because it reflects the flow of the data receivers -> processors -> exporters VERB by service